### DISCUSSION TITLE
FalconPy and put-and-run

### DISCUSSION THREAD

**Discussion by jsdjjsd** [CrowdStrike/falconpy#594] [Q&A] [answered]
Hi @jshcodes 

Any plans on implementing put-and-run in FalconPy? Quite a time-saver when scripting. ;)

Best regards,
jsd

**Comment by jshcodes** [comment, marked as answer]
Hi @jsdjjsd -

We're working on advanced RTR functionality now!  W00t!  This will be part of the [Caracara](https://github.com/CrowdStrike/Caracara) project, which should start showing some updates in the repo over the next couple of weeks. (The version currently posted is an older alpha.)  

I'll post a note here once we've gotten the latest code base pushed up to the repo. 😃

**Reply by jshcodes** [reply]
Hi @jsdjjsd -

Latest version of [Caracara](https://github.com/CrowdStrike/caracara) has merged (with v.0.1.2 on deck to merge very soon). 

Check it out and let us know what else you'd like to see! 😄

**Reply by jsdjjsd** [reply]
> Hi @jsdjjsd -
> 
> Latest version of [Caracara](https://github.com/CrowdStrike/caracara) has merged (with v.0.1.2 on deck to merge very soon).
> 
> Check it out and let us know what else you'd like to see! 😄

Hi @jshcodes 

Nice! Thanks for letting me know. I'll make sure to check it out. :)
